Original patch from: gentoo/src/patchsets/glibc/2.9/6017_all_alpha-glibc-2.8-creat.patch

-= BEGIN original header =-
alpha does not have a __NR_creat

http://bugs.gentoo.org/227275
http://sourceware.org/bugzilla/show_bug.cgi?id=6650

-= END original header =-

diff -durN glibc-2_9.orig/sysdeps/unix/sysv/linux/wordsize-64/creat64.c glibc-2_9/sysdeps/unix/sysv/linux/wordsize-64/creat64.c
--- glibc-2_9.orig/sysdeps/unix/sysv/linux/wordsize-64/creat64.c	2007-11-10 20:34:26.000000000 +0100
+++ glibc-2_9/sysdeps/unix/sysv/linux/wordsize-64/creat64.c	2009-02-02 22:01:35.000000000 +0100
@@ -1 +1,5 @@
 /* Defined as alias for the syscall.  */
+#include <sysdep.h>
+#ifndef __NR_creat
+#include "../../../../../io/creat64.c"
+#endif
